### DISCUSSION TITLE
How do I use a FragmentCustom instead of using ActivityCustom

### DISCUSSION THREAD

**Discussion by ArleyPereira** [journeyapps/zxing-android-embedded#687] [Q&A]
Which library version are you using? 4.3.0

I would like to use a FragmentCustom instead of an ActivityCustom. It is possible, could you help me?

**Comment by rkistner** [comment]
Could you give more details on what you are trying to achieve?

There are a couple of different examples in the repository which you can look at. This one embeds a BarcodeView inside a Fragment, which is perhaps relevant?

https://github.com/journeyapps/zxing-android-embedded/blob/master/sample/src/main/java/example/zxing/TabbedScanning.java

**Comment by ArleyPereira** [comment]
Sorry, I couldn't use the example above. I have tested it but it is not reading the barcode

I'm using a navigation chart, so I need to use a FragmentCustom instead of an ActivityCustom.

how can i use findNavController().navigate(R.id....) informing a:

```
val options = ScanOptions()
    .setDesiredBarcodeFormats(ScanOptions.ITF)
    .setOrientationLocked(true)
    .setCaptureActivity(FragmentCustom::class.java)
     barcodeLauncher.launch(options)
```
